recipes/default.rb

```diff
@@ -4,10 +4,13 @@
 
 pkgs = %w[
   glibc.i686
-  glibc
+  glibc.x86_64
+  libgss.x86_64
```

Comment:
libgss? is this really necessary? do you mean `libgcc.x86_64`?

Reply:
doh

Reply:
fixed

Reply:
yeah this is - when running the recipe you get errors with the i686 package version not being in sync with the x86_64 package

Reply:
can you try instead:
```ruby
pkgs = %w[
  glibc
  libgcc
]
yum_package pkgs do
  arch ['x86_64', 'i686']
end

yum_package ['compat-libstdc++-33', 'compat-db']
```

Reply:
@sandstheman ^^